plugin/pkg/auth/authorizer/webhook/webhook_test.go

```diff
@@ -508,33 +524,77 @@ func TestWebhookCache(t *testing.T) {
 	attr := authorizer.AttributesRecord{User: &user.DefaultInfo{Name: "alice"}}
 	serv.allow = true
 	serv.statusCode = 500
-	if err := wh.Authorize(attr); err == nil {
+	authorized, _, err := wh.Authorize(attr)
```

Comment:
When possible prefer table driven tests[0] instead of rewriting these over and over again. 

[0] https://github.com/golang/go/wiki/TableDrivenTests